django/db/models.py

```diff
@@ -226,18 +222,19 @@ class FieldInformation(models.Model):
     manager                 = FieldsManager()
     objects                 = models.Manager() # default manager
     
-    table                   = _CHAR(max_length=35, **_NULLOKSTR)
-    field                   = _CHAR(max_length=35, **_NULLOKSTR)
-    alias                   = _CHAR(max_length=35, **_NULLOKSTR)
-    queryset                = _CHAR(max_length=35, **_NULLOKSTR)
+    table                   = _TEXT(**_NULLOKSTR)
```

Comment:
It's frustrating that Django doesn't support Postgres's ability to define varchar fields with no length limit. That would let you do _CHAR(**_NULLOKSTR) which is what you really want here. No change needed, just venting!